README.md

```diff
@@ -1,16 +1,6 @@
 # Introduction
 
-Carmen is a fast and space conservative database for blockchains. It outperforms other projects in the
```

Comment:
I think this two original lines should be (in some form) kept. It is our pitch - the speed that outperforms others and the space is a fraction of what provides others. 

The new pitch is way too complicated, it suddenly speaks about balances, code, smart contract.

Reply:
@kjezek  - can you rewrite it and submit it? I can check after.

Reply:
done, please have a look

Reply:
We should focus on functionality rather than non-functional properties.